ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/block/BlockProcessor.java

```diff
@@ -177,6 +179,10 @@ void processExecutionLayerWithdrawalRequests(
       final Supplier<ValidatorExitContext> validatorExitContextSupplier)
       throws BlockProcessingException;
 
+  void processConsolidations(
+      MutableBeaconState state, final SszList<SignedConsolidation> consolidations)
```

Comment:
we dont usually go final on interfaces, but either way consistency probably good.... both final or both not...